### PR TITLE
More ex_act changes

### PR DESCRIPTION
**Pull request by Stingray540**
Did I do it XO?

:cl: Stingray
rscadd: Adds a system to allow each species to have variable reactions to ear_deaf/damage and confusion from explosions. As of now, every species bar Spartans/Sangheili/Brutes (who have a 0.5 modifier) have a 1.0 default modifier.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/species.dm [hunk 1]
```diff
@@ -125,6 +125,8 @@
 		"Your chilly flesh stands out in goosebumps."
 		)
 
+	var/explosion_affect_mod = 1.0 //Modifier on how much a species can resist explosive sideeffects
```
Comment (X0-11) [inline code/modules/mob/living/carbon/human/species/species.dm]: Effect is probably the right one here

**Review by X0-11 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species/species.dm (hunk 1 above)
Comment (Stingray540) [reply code/modules/mob/living/carbon/human/species/species.dm]: Resolved in new commit.

**Review by X0-11 — APPROVED** [review]

**Comment by X0-11** [comment]
whilst I admire your honesty in commit names, you don't have to label everything done due to a code review as "appeasing XO"

**Comment by Stingray540** [comment]
At least it makes a change from "Appeases Travis"

**Pull request merged**